libs/leonsans/src/core/model.ts

```diff
@@ -0,0 +1,282 @@
+import { getTypo } from '../font/index';
+import { getAlignGapX, setAlignGapX } from './align';
+import { getTextGroup } from './group';
+import { getGrid, getGuide } from './guide';
+import { getLengths } from './length';
+import { getPaths } from './paths';
+import {
+  addRectToPaths,
+  getCenter,
+  getCircleRound,
+  getFontRatio,
+  getFontW,
+  getLeading,
+  getLineW,
+  getLines,
+  getRange,
+  getScale,
+  getScaledRect,
+  getTracking,
+  getWeightRatio,
+} from './util';
+
+export class Model {
+  lineWidth_: number;
+  drawing_: Drawing[];
+  data_: ModelData[];
+  rect_: Rect;
+  align_: Align;
+  scale_: number;
+  fontRatio_: number;
```

Comment:
```suggestion
  lineWidth_: number;
  drawing_: Drawing[]; // Progress of 
  data_: ModelData[]; // Stores all information needed to render Leon Sans
  rect_: Rect; // Position and size of rectangle area that Leon Sans occupies
  align_: Align;
  scale_: number;
  fontRatio_: number; // Aspect ratio of text determined font weight (맞나?)
```

애매한 프로퍼티에 주석을 달아봤으요. 요것들만 알아도 코드 이해가 쉬울 것 같슴다. 근데 영어로 단 주석이랑 한글로 단 주석의 기준이 있으심?